src/main/java/core/basesyntax/figure/FigureSupplier.java

```diff
@@ -0,0 +1,46 @@
+package core.basesyntax.figure;
+
+import java.util.Random;
+
+public class FigureSupplier {
+
+    private Random random = new Random();
+    private ColorSupplier colorSupplier = new ColorSupplier();
+
+    public Figure getRandomFigure() {
+
+        int figureType = random.nextInt(5);
```

Comment:
Magic numbers like '5' should be replaced with named constants to improve code readability and maintainability.

Reply:
Fix the comment